response_operations_ui/views/collection_exercise.py

```diff
@@ -57,8 +72,10 @@ def view_collection_exercise(short_name, period, error=None, success_panel=None)
                            collection_instruments=ce_details['collection_instruments'],
                            eq_ci_selectors=ce_details['eq_ci_selectors'],
                            error=error,
+                           error_panel=error_panel,
```

Comment:
Doesn't look like error_panel is being used?

Reply:
I'm refactoring this in my PR so this will change 👍